Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -223,6 +230,18 @@ - (void)updateDrawerPosition:(NSString *)positionString
     [self.bottomSheetVC setDrawerPositionWithPosition:position animated:YES completion:nil];
 }
 
+- (void)updateSafeAreaInsets
+{
+    UIEdgeInsets safeAreaInsets = self.view.safeAreaInsets;
+    [self.mapVC setProperty:@{
+                              @"top": [NSNumber numberWithFloat:safeAreaInsets.top],
+                              @"bottom": [NSNumber numberWithFloat:safeAreaInsets.bottom],
+                              @"left": [NSNumber numberWithFloat:safeAreaInsets.left],
+                              @"right": [NSNumber numberWithFloat:safeAreaInsets.right]
```

Comment:
Not a big deal, but we can use ObjC object literals here:

```diff
- @"right": [NSNumber numberWithFloat:safeAreaInsets.right]
+ @"right": @(safeAreaInsets.right)
```

Reply:
True, forgot about that